src/views/CreateInstance.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <main v-if="formState === 0" class="create-Instance">
+    <ViewHeader
+      :title="`Choose a resource type for the new instance`"
```

Comment:
This does not need to be a template string, and `:` can be removed, as you have no variable binding here.

Reply:
Please use consistent upper/lower case for `instance`. In the next step, instance is written in upper case

Reply:
done